libraries/primitive_consul_lock.rb

```diff
@@ -66,12 +67,18 @@ def backoff
     def wait_until(action, opts = {})
       dc = "(in #{@options[:datacenter]})" if @options[:datacenter]
       Chef::Log.info "Will #{action} the lock #{path} #{dc}"
+
+      start_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
```

Comment:
use Time.now instead. It is easier to read and I don't think there is a use case to get a monotonic value here (or there should be a comment explaining exactly why)